internal/commands/parse.go

```diff
@@ -41,13 +42,8 @@ func NewParseCommand(ctx context.Context) *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, fileList []string) error {
-			params := &runner.ParseParams{}
-			viper.Unmarshal(params)
-			runner := runner.ParseRunner{
-				Params:        params,
-				ConfigManager: &parser.ConfigManager{},
-			}
-			out, err := runner.Run(ctx, fileList)
```

Comment:
I agree with more complicated commands, like `test`, but using this pattern on `parse` felt slightly overkill. The `parse` command is _really_ just the `marshal` function inside of `runner/parse.go`. Theres some small decision making based on the `combine` flag or not.

IMO we don't really need to mock/fake out `GetConfigurations`, because with Go we can just new up our own configuration and pass it along to the private methods. I didn't really change the tests all that much, and was able to remove extra structs/interfaces required to fake the `GetConfigurations` method.

I'm not super opinionated either way, but it just felt like a lot of extra abstraction to test something that I don't think provides a lot of value.